src/signature.ts

```diff
@@ -58,3 +58,13 @@ export class Signature {
         return this.valueHex;
     }
 }
+
+export function interpretSignatureAsBuffer(signature: { hex(): string; } | Uint8Array): Buffer {
```

Comment:
Is the returning type necessary? Seems inferred.

Reply:
By convention, on `sdk-core` we add return types (usually).